go.mod

```diff
@@ -2,13 +2,14 @@ module github.com/icinga/icinga-notifications
 
 go 1.24.0
 
+toolchain go1.24.6
+
 require (
 	github.com/creasty/defaults v1.8.0
 	github.com/emersion/go-sasl v0.0.0-20241020182733-b788ff22d5a6
 	github.com/emersion/go-smtp v0.24.0
 	github.com/google/uuid v1.6.0
-	github.com/hashicorp/golang-lru/v2 v2.0.7
-	github.com/icinga/icinga-go-library v0.7.2
+	github.com/icinga/icinga-go-library v0.7.3-0.20251022120618-6600889adc38
```

Comment:
That commit (https://github.com/Icinga/icinga-go-library/commit/6600889adc38ffe2088948411c852cb387b5762f) is included in the `main` branch of that repo, so it'll be included in the next release. Pinning a commit should be fine for the meantime, but for a tagged version in this repo, a tagged version from that repo should be used (I'll create an issue so that we won't forget).

Reply:
> I'll create an issue so that we won't forget

Done: #347